packages/core/src/runtime-metadata.feature.ts

```diff
@@ -1,9 +1,17 @@
 import { Config } from './entities/config';
 import { Feature } from './entities/feature';
 
+export interface FoundFeatures {
+    featureName: string;
+    configurations: string[];
+}
+
 export interface RuntimeMetadataConfig {
     devport?: number;
-    applicationPath: string;
+    applicationPath?: string;
+    featureName?: string;
+    foundFeatures?: FoundFeatures[];
+    isWorkspace?: boolean;
```

Comment:
the only thing I think here is maybe add this config to a key called "engineerMetadata" or something like that? to be clear it's dev mode only data

Reply:
Renamed config to `engineerMetadataConfig`